test/unit-test/test_workflow_status.c

```diff
@@ -26,19 +26,22 @@
 
 const char* _msg = "message foo";
 
-static component_t _component;
+component_t* __real_status_create(
+    const char* text,
+    bool status_success,
+    int delay,
+    void (*callback)(void*),
+    void* callback_param);
```

Comment:
I'm a bit confused and surely missing somehing. Doesn't `status_create` take only 2 args now and shouldn't this be:

```c
component_t* __real_status_create(const char* text, bool status_success);
```
?

OTOH the CI tests pass. So, this must be working somehow?

Reply:
Wow, that's messed up. I don't know why this compiles. Not even the memory sanitizer complains.

Reply:
Right. This wasn't even supposed to compile. Maybe I'll try to figure it out in my local copy.